core/src/main/java/org/axonframework/eventhandling/AbstractEventProcessor.java

```diff
@@ -133,7 +138,8 @@ protected void processInUnitOfWork(List<? extends EventMessage<?>> eventMessages
                                        Segment segment) throws Exception {
         try {
             unitOfWork.executeWithResult(() -> {
-                MessageMonitor.MonitorCallback monitorCallback = messageMonitor.onMessageIngested(unitOfWork.getMessage());
+                MessageMonitor.MonitorCallback monitorCallback = messageMonitor.onMessageIngested(unitOfWork
```

Comment:
Nit: Probably a follow up from the reindent hotkey, but I'd like suggestiong moving the part after the `=` to a following line i.o. cutting of the `unitOfWork.getMessage()` call.